extension/main.js

```diff
@@ -13,6 +13,15 @@
 		chrome.browserAction.setTitle({
 			title: title
 		});
+
+		if (badge && window.GitHubNotify.settings.get('showDesktopNotif')) {
```

Comment:
@inikulin It should be easier now because we set the current "count" value in storage: https://github.com/sindresorhus/github-notifier-chrome/blob/master/extension/main.js#L57